hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+{% extends "pages/genericresource.html" %}
+{% load geoanalytics_tags pages_tags staticfiles mezzanine_tags crispy_forms_tags hydroshare_tags%}
+
+{% block extra_css %}
+    {{ block.super }}
+    <link rel="stylesheet" href='{% static "css/jquery.dataTables.min.css" %}'/>
+{% endblock %}
+
+{# override content_block in generice.html#}
+{% block content_block %}
+    <hr>
+    {# collection content table #}
+    <div id="collection_content_div">
+        <div id="title_and_add_btn_div">
+           <h3 style="display: inline">Collection Contents </h3>
+           {% if edit_mode %}
+               <a type="button" class="btn btn-success" onclick="$('#collection-candidate').modal('show');" >
+                   <span class="glyphicon glyphicon-plus"><span class="button-label"> Add resources...</span></span>
+               </a>
+
+           {% endif %}
+        </div>
+        <div id="collection-table-div">
+            {% if edit_mode %}
+                <form id="collector-new" name="collector-new" action="/hsapi/_internal/{{collection_res_id}}/update-collection/" method="POST">
+            {% endif %}
+
+            <table id="collection-table" class="table-hover table-striped" width="100%">
+                <thead>
+                <tr>
+                    <th>Add</th>
+                    <th>Title</th>
+                    <th>Type</th>
+                    <th>Owners</th>
+                    <th>Sharing status</th>
+                    <th>My Permission</th>
+                    <th>Remove</th>
+                </tr>
+                </thead>
+                <tbody>
+                 {% if collection %}
+                    {% for res in collection %}
+                        <tr id="{{ res.short_id }}">
+                            <td>
+                                <input class="row-selector" type="checkbox" id="{{ res.short_id }}">
+                            </td>
+                            <td>
+                                {% if res|user_permission:request.user.pk|lower == "none" %}
+                                    <strong>{{ res.metadata.title }}</strong>
+                                {% else %}
+                                    <strong><a href="{{ res.get_absolute_url }}" target="_blank">{{ res.metadata.title }}</a></strong>
+                                {% endif %}
+                            </td>
+                            <td>
+                                {{ res.resource_type }}
+                            </td>
+                            <td>
+                                {% for owner in res.raccess.owners.all %}
+                                    {% if forloop.counter0 > 0 %} · {% endif %}
```

Comment:
This empty if-endif block does not appear to serve any purpose. Can it be removed?